integrations/ethereum.go

```diff
@@ -0,0 +1,92 @@
+package integrations
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/stellar/starbridge/model"
+)
+
+// FetchEthTxByHash returns a model.Transaction
+func FetchEthTxByHash(txHash string) (*model.Transaction, error) {
```

Comment:
uses an infura node that I've setup. Hardcoded API key for now but this is all temporary and all hardcoded things will be removed within a week